logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -69,4 +71,13 @@ class UserConfigDataSource(
         wrapStorageRequest {
             userConfigStorage.isConferenceCallingEnabled()
         }
+
+    override fun isReadReceiptsEnabled(): Flow<Either<StorageFailure, Boolean>> =
+        userConfigStorage.isReadReceiptsEnabled().wrapStorageRequest()
+
+
+    override fun setReadReceiptsEnabled(enabled: Boolean): Either<StorageFailure, Unit> =
+        wrapStorageRequest {
+            userConfigStorage.persistReadReceipts(enabled)
```

Comment:
Is there a reason to have read receipts only changed locally and to synced with other clients of the user

Reply:
Good catch, I'm implementing this now :)